webapp-python/build-tools/python-build-local.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+# Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+#
+#This helper script will package the Python project in
+# webapp-python, build it locally, package it as a zip
+# file for Lambda, and upload that zip file to an S3 bucket.
+#
+#Prerequisites:
+# 1. Python 3.6 available locally.
```

Comment:
3.6, or 3.6+?

Reply:
3.6 specifically, because that is the version that is available in the Lambda runtime.

Reply:
Oh. Duh.